pkg/ansible/controller/controller.go

```diff
@@ -58,6 +59,7 @@ func Add(mgr manager.Manager, options Options) {
 		Runner:          options.Runner,
 		EventHandlers:   eventHandlers,
 		ReconcilePeriod: options.ReconcilePeriod,
+		ManageStatus:   options.ManageStatus,
```

Comment:
Looks like the indentation drifted. I have a vim plugin that runs "go fmt" on every save, which is handy for preventing that sort of thing.